docs/blog/v5-intro.md

```diff
@@ -0,0 +1,167 @@
+
+### Multi grafana instances support
+
+This is one of our most requested and [oldest issues](https://github.com/grafana-operator/grafana-operator/issues/174) in the grafana-operator and we have finally been able to provide it.
+
+In v5 you can now run multiple instances of grafana in the same namespace and externally.
+
+### External grafana instances
+
+It's now possible to configure external grafana instances through the operator.
+So if you are using grafana cloud or a hosted grafana instances in one of the big cloud providers you can still use gitops to apply your dashboards to them.
+
+One of the limitations with the external grafana instances is that we don't support installing plugins.
+If your dashboard is using a plugin we assume that you have manually installed it to your grafana instance.
+
+### Grafana config
+
+The deployment configuration is more **opinionated** in it's default setup while at the same time giving you the user access to the whole deployment spec so you can overwrite anything you want.
+For example when you start a grafana instance it will by default have decent security defaults, it will set `RunAsNonRoot` on your grafana instance when they get created.
+But since you have access to the whole deployment spec you can always overwrite these kind of settings.
+
+The same thing applies to all our resources, for example ingress.
+The bad thing here is of course that we won't hide any of the logic that made the operator "easy" to get started with.
+So if you want to configure an ingress for your grafana instance you will have to define it just like any other ingress resources.
+But it makes it extremely extensible and hopefully this will make the operator support any setup you want.
+
+### Grafana ini
+
+Just like deployment config the same thing goes for grafana config, instead of having a CRD that contains all the grafana ini values we have instead made it possible to write any value as you want.
+The **bad** thing is of course that you as an end user don't get any help with verifying types etc.
+The **good** thing is that you can set any value you want, if you want to try out an experimental feature you can do so without first having to create a PR to the operator.
+
+### Usage of status fields
+
+The usability of v4 have more then once been laking. It's been really hard to understand why something is happening and you more or less have been forces to go in and look at the logs.
+This is something that we wanted to change with v5.
+
+In v5 you will be able to look at your grafana dashboard CR and see that it got a matching grafana instance (the label selector is correct) and if it's been applied to the grafana instances.
+
+### No more plugin sidecar
+
+As you might know we have had a python container that have been an [sidecar](https://github.com/grafana-operator/grafana_plugins_init) to the operator that have been in charge of installing the plugins to grafana.
```

Comment:
For future proofing, refer to a tag version of this repo if the plan is for v5 to be merged into this repo?

Reply:
So I'm a bit unsure what you meant, but in the top of the blog I have now added a mention on how we will manage the branches in the future. Version 5 will be the new mater and we will keep version 4 as a branch.

About grafana_plugins_init repo we will archive it when we stop supporting version 4 all together.
I hope this makes it clearer.